### DISCUSSION TITLE
Multiple tables inside a worksheet

### DISCUSSION THREAD

**Discussion by zadptor** [ozlerhakan/poiji#258] [Q&A] [answered]
Hello, Is there a workaround solution for reading excel worksheet with multiple tables inside it? For example, I have an excel file with worksheet that have tableA, tableB and tableC. in different location for example. how can I read only tableA?

**Comment by ozlerhakan** [comment, marked as answer]
Hi @zadptor , Poiji doesn't have this option in one shot.  Since this request is a more specific use, I can only advise you to first split the  tables in to different sheets and then iterate each sheet to form all tables.

**Reply by zadptor** [reply]
I actually tried something similar. Actually it works with simple text, but not with formulas. The formulas only work if we read the workbook from a file or inputstream right? correct me if I am wrong.

**Reply by ozlerhakan** [reply]
It should normally work with any case. You can give me a test excel file with formulas to check If you want.

**Reply by zadptor** [reply]
Thank you for your prompt reply. Unfortunately I cant provide sample of excel file because of NDA with my client. appreciate your help. thanks a lot
